the-sz.Yale version 1.21

the-sz opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20991)

the-sz committed:
the-sz.Yale version 1.21

Trenly commented on manifests/t/the-sz/Yale/1.21/the-sz.Yale.yaml:
```diff
@@ -0,0 +1,23 @@
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://the-sz.com/services/winget/get.php?p=yale
```
```suggestion
  InstallerUrl: https://the-sz.com/services/winget/Setup.exe
```

the-sz replied:
to distinguish between different products I would like to keep the php download script in between

Trenly commented on manifests/t/the-sz/Yale/1.21/the-sz.Yale.yaml:
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.Yale
+PackageVersion: 1.21
```
The package version does not appear in the ARP entry for this version. This will cause an upgrade loop
![image](https://user-images.githubusercontent.com/12611259/125690363-3b003c33-74a0-4d2c-add9-442dfd2532d9.png)

the-sz replied:
fixed with new commit

Trenly commented on manifests/t/the-sz/Yale/1.21/the-sz.Yale.yaml:
```diff
@@ -0,0 +1,23 @@
+PackageUrl: https://the-sz.com/products/yale/
+License: Copyright (c) the-sz.com
+LicenseUrl: https://the-sz.com/products/license.php
+ShortDescription: Bring back the Classic Windows XP Start Menu to Windows 10. With one click you have access to all of your Start Menu entries. Plus direct access to the Control Panel, Administrator command line and Run dialog.
```
This description doesn't seem accurate to the package

the-sz replied:
I'm sorry, my yaml creation script had an error :-(

palenshus commented:
/azp run

OfficialEsco reviewed (changes requested):
https://github.com/microsoft/winget-pkgs/pull/20991/files#r669942369
Download URL: Since its a static link anyways i guess its okay
Version: Hopefully winget-cli gets a workaround soon, or a user of their software should complain
Description: This needs to change

the-sz committed:
Update the-sz.Yale.yaml

the-sz commented:
> Possible duplicate package entry.
> 
> Similar installer SHA256 hash found in manifest:
> 
> * `InstallerSha256: b1099c8188eec1b07a609c6027edba3a46b7982ddb12a2b4d16a3fbe6dbc820a`
>   
>   * manifests\t\the-sz\Spencer\1.27
>   * manifests\t\the-sz\Bear\1.60

False positive, installer downloads appropriate application based on command line option

the-sz commented:
> https://github.com/microsoft/winget-pkgs/pull/20991/files#r669942369
> Download URL: Since its a static link anyways i guess its okay
> Version: Hopefully winget-cli gets a workaround soon, or a user of their software should complain
> Description: This needs to change

Version: fixed
Description: fixed. I'm sorry!
